test/e2e-image/e2e.sh

```diff
@@ -28,8 +28,9 @@ ginkgo_args=(
   "-randomize-all"
   "-flake-attempts=2"
   "-fail-fast"
-  "-progress"
-  "-slow-spec-threshold=${SLOW_E2E_THRESHOLD}"
+  "--show-node-events"
+  "--poll-progress-after=180s"
+#  "-slow-spec-threshold=${SLOW_E2E_THRESHOLD}"
```

Comment:
why is this commented out? https://onsi.github.io/ginkgo/MIGRATING_TO_V2#changed--slowspecthreshold this is still supported right?